templates/kubeconfig.tpl

```diff
@@ -36,3 +36,4 @@ users:
           value: ${v}
   %{~ endfor ~}
 %{ endif ~}
+${""}
```

Comment:
It would be cleaner to just remove the ~ from the end of that last `endif` statement. That is what is removing the terminating whitespace.

Reply:
Done.